modules/ROOT/pages/standalone-mule-setup.adoc

```diff
@@ -0,0 +1,36 @@
+= Standalone Mule Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Verify Supported Standalone Mule Versions
+
+Anypoint Visualizer supports the following Mule versions for standalone Mule deployments:
+
+* Mule 3.8.7
+* Mule 3.9.1+
+* Mule 4.1.x
+* Mule 4.2.x
+
+== Install Agent
+
+To support Anypoint Visualizer for standalone Mule deployments, install the xref:monitoring::am-install.adoc[Anypoint Monitoring agent].
+Anypoint Visualizer also requires an additional header injection for standalone Mule deployments.
+However, due to a potential performance impact for Mule 4, header injection is disabled by default. 
+
+== Enable Header Injection
+
+For a standalone Mule deployment, you must enable the header injection at the server (not app) level.
+This affects all applications running on the server.
+
+. To enable the header injections for apps deployed to a standalone Mule, click *Server* in the menu on the left of *Runtime Manager*.
+. Select the server for which to enable the header injection, and click *Manage Server*.
+. Click the Properties tab, and set the following property in the server:
+`anypoint.platform.config.analytics.agent.header_injection.disabled=false`.
+You must set the property to `false` as shown in the example to enable header injection, since the property is disabled by default.
```

Comment:
```suggestion
You must set the property to `false`, as shown in the example to enable header injection, because the property is disabled by default.
```